src/create/after-create.js

```diff
@@ -0,0 +1,10 @@
+/* eslint global-require:"off" */
+var runPluginFunction = require('../util/runPluginFunction')
+/**
+ * calls any plugins registered in .arc with afterCreate
+ */
+module.exports = function afterCreate(params, callback) {
```

Comment:
I'm not a huge fan of mixing callbacks and promises, so I made the `util/runPluginFunction` always return a promise and here we just wait for it to be complete to call the original callback.